DailyQuickstart.xcodeproj/project.pbxproj

```diff
@@ -17,7 +17,7 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXFileReference section */
-		221F387E29C64D7B008B8A6B /* Daily_iOS_Quickstart.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = Daily_iOS_Quickstart.app; sourceTree = BUILT_PRODUCTS_DIR; };
+		221F387E29C64D7B008B8A6B /* DailyQuickstart.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = DailyQuickstart.app; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
App names do not typically contain underscores, and we haven't include iOS in the name for other example code, so I thought we should be consistent here too.